reference-architecture/gce-cli/config.yaml.example

```diff
@@ -84,3 +84,6 @@ master_boot_disk_size: 35
 node_boot_disk_size: 25
 node_docker_disk_size: 25
 node_openshift_disk_size: 50
+
+# Custom VPC Subnet, example value: '10.160.0.0/20'
+gce_vpc_custom_subnet_cidr: ''
```

Comment:
If the variable is empty as the default value, it will create a new subnet, am I right? (I didn't even deploy a thing in GCP yet)
If that's the case, I'd say to add an explanation as "if it is empty, it will create a new subnet with this address a.b.c.d/x"